src/plugins/intel_gpu/src/graph/impls/ocl_new/rope_opt.cpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (C) 2025 Intel Corporation
```

Comment:
[random spot]
Maybe we should move cl kernels and impls to dedicated operation specific sub-folders?

Reply:
I also though about it, not sure which approach will be better. Having sub-folders looks more clear from the structure perspective, but with that we just duplicate operation name in folder name and file name. Omitting op prefix in file names is the sub-folder may lead to difficulties with required file search in some IDEs, so I think we can't avoid the duplication with sub-folders approach.